pyproject.toml

```diff
@@ -75,6 +70,37 @@ spyglass_cli = "spyglass.cli:cli"
 
 [project.optional-dependencies]
 position = ["ffmpeg", "numba>=0.54", "deeplabcut<2.3.0"]
+test = [
+    "pytest",         # unit testing
+    "pytest-cov",     # code coverage
+    "kachery",        # database access
+    "kachery-client",
+    "kachery-cloud",
+]
+docs = [
+    "mike",                  # Docs versioning
+    "mkdocs",                # Docs core
+    "mkdocs-exclude",        # Docs exclude files
+    "mkdocs-exclude-search", # Docs exclude files in search
+    "mkdocs-gen-files",      # Docs API generator
+    "mkdocs-jupyter",        # Docs render notebooks
+    "mkdocs-literate-nav",   # Dynamic page list for API docs
+    "mkdocs-material",       # Docs theme
+    "mkdocstrings[python]",  # Docs API docstrings
+]
+
+[tool.hatch.version]
+source = "vcs"
+
+[tool.hatch.build.hooks.vcs]
+version-file = "src/spyglass/_version.py"
```

Comment:
```suggestion
# this file is created/updated when the package is installed and used in
# src/spyglass/__init__.py to set `spyglass.__version__`
version-file = "src/spyglass/_version.py"
```

Adding a comment since this behavior is a little unintuitive for now.